client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-page-preview.scss

```diff
@@ -0,0 +1,42 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+@import "@automattic/typography/styles/variables";
+
+$font-family: "SF Pro Text", $sans;
+
+.pattern-assembler__preview {
+	flex-basis: 300px;
+	flex-shrink: 0;
+
+	.pattern-assembler__preview-frame {
+		background: #f8f8f8;
+		border: 10px solid #f8f8f8;
+		border-radius: 13px;  /* stylelint-disable-line scales/radii */
+		box-shadow:
+			0 5.25469px 5.25469px 0 rgba(0, 0, 0, 0.02),
+			0 11.38516px 8.75781px 0 rgba(0, 0, 0, 0.03);
+		height: 350px;
+		overflow: hidden;
+		position: relative;
+
+		&-content {
+			border-radius: 8.758px;  /* stylelint-disable-line scales/radii */
+			bottom: 0;
+			left: 0;
+			overflow: scroll;
```

Comment:
it might be better to specify the `overflow-y` as the horizontal scrollbar shows by default (if you set up the system settings) and it doesn't look good

![image](https://github.com/Automattic/wp-calypso/assets/13596067/eccbdd75-5e78-44c7-ae6e-560c95db648e)

Reply:
Good point! I should start enabling scrollbars by default.